index.js

```diff
@@ -0,0 +1,17 @@
+var path = require('path');
+
+var tabzillaEntryPoint = require.resolve('mozilla-tabzilla');
+var tabzillaDir = path.join(path.dirname(tabzillaEntryPoint), 'css');
+
+function includePaths() {
+  return tabzillaDir;
+}
+
+module.exports = {
+  includePaths: includePaths(),
+  with: function() {
+    var paths  = Array.prototype.slice.call(arguments);
+    var result = [].concat.apply(includePaths(), paths);
```

Comment:
Nit: Is there any reason we cannot simply use an anonymous function here instead of the named function?